ui/partials/main/thFilterPanel.html

```diff
@@ -1,56 +0,0 @@
-<div id="filter"
-     class="th-top-nav-options-panel model-body"
-     ng-controller="FilterPanelCtrl">
-  <div class="nav-panel-help-text">Specify which jobs to show based on this filter criteria.
-    <span class="pull-right">
-      <span class="btn btn-default btn-xs"
-            title="Pin all jobs that pass the global filters"
-            ng-click="pinAllShownJobs()"><i class="glyphicon glyphicon-pushpin"></i> pin all showing</span>
-      <span class="btn btn-default btn-xs"
-            title="Show only superseded jobs"
-            ng-click="thJobFilters.setOnlySuperseded()">superseded only</span>
-      <span class="btn btn-default btn-xs"
-            title="Reset to default status filters"
-            ng-click="thJobFilters.resetNonFieldFilters()">reset</span>
-    </span>
-  </div>
-
-  <!-- Result status filters -->
-  <span ng-repeat="group in filterGroups"
-        class="panel panel-default th-option-group th-inline-option-group">
-
-    <!-- Group heading -->
-    <div class="panel-heading th-option-heading">{{::group.name}}
```

Comment:
The `.th-top-nav-options-panel .th-option-heading` CSS rule in `treeherder-navbar-panels.css` seems unused after this